specs/withdrawals.md

```diff
@@ -55,18 +55,27 @@ We first describe the end to end flow of initiating and finalizing a withdrawal:
 ### On L2
 
 An L2 account sends a withdrawal message (and possibly also ETH) to the `L2ToL1MessagePasser` predeploy contract.
-   This is a very simple contract that stores the a hash of the withdrawal data.
+This is a very simple contract that stores the a hash of the withdrawal data.
 
 ### On L1
 
```

Comment:
Slight nit, markdown lists should always use `1` and the renderer will handle incrementing at runtime. This is to reduce diffs when making modifications